standard/lexical-structure.md

```diff
@@ -927,6 +938,7 @@ Each string literal does not necessarily result in a new string instance. When t
 
 > *Example*: For instance, the output produced by
 >
+> <!-- Example: {template:"standalone-console",name:"ObjectReferenceEquality",expectedOutput:["True"]} -->
```

Comment:
Note to self: we're currently inconsistent about whether to include spaces between metadata values.
Let's not *try* to get consistency yet; I can easily write tooling to make everything consistent later.